containerd-shim-v2/service.go

```diff
@@ -72,13 +73,31 @@ func New(ctx context.Context, id string, publisher events.Publisher) (cdshim.Shi
 	vci.SetLogger(ctx, logger)
 	katautils.SetLogger(ctx, logger, logger.Logger.Level)
 
+	// load runtime config so that tracing can start if enabled
+	_, runtimeConfig, err := katautils.LoadConfiguration("", false, true)
+	if err != nil {
+		return nil, err
+	}
+
+	// create tracer
+	var tracer opentracing.Tracer
+	tracer, err = katautils.CreateTracer("kata")
+	if err != nil {
+		return nil, err
+	}
+	// create span
+	span, ctx := trace(ctx, "New")
+	defer span.Finish()
+
 	ctx, cancel := context.WithCancel(ctx)
 
 	s := &service{
 		id:         id,
 		pid:        uint32(os.Getpid()),
 		ctx:        ctx,
+		tracer:     tracer,
```

Comment:
@cmaf Could you help me understand how this Tracer instance is used?  I'm probably overlooking something but I don't see it referenced anywhere.

Reply:
@pmores, thanks for catching this. You're not overlooking anything, I previously used `s.tracer` in the code but that had been removed. I removed this instance.

Reply:
@cmaf Thanks!